makefile

```diff
@@ -111,7 +111,7 @@ test-api-functional: reset-test-database
 	@NODE_ENV=test NODE_PORT=3010 ./node_modules/.bin/mocha --require "./babel-transformer" --require=co-mocha --recursive ./e2e/api
 
 test-frontend-unit:
-	@NODE_ENV=test ./node_modules/.bin/mocha --compilers="css:./webpack/null-compiler,js:babel-core/register" --recursive ./src/frontend/js/**/*.spec.js
+	@NODE_ENV=test ./node_modules/.bin/mocha --require ./test/setupJsdom.js --compilers="css:./webpack/null-compiler,js:babel-core/register" --recursive ./src/frontend/js/**/*.spec.js
```

Comment:
I wonder if you can't enable it on a per-test basis, instead of for the entire suite.